member-service/src/main/java/com/hedvig/memberservice/jobs/BankidMetrics.kt

```diff
@@ -0,0 +1,40 @@
+package com.hedvig.memberservice.jobs
+
+import io.micrometer.core.instrument.MeterRegistry
+import org.springframework.stereotype.Component
+
+@Component
+class BankidMetrics(
```

Comment:
```suggestion
class SwedishBankidMetrics(
```

Reply:
Also, small I on Id/ID looks weird
```suggestion
class SwedishBankIdMetrics(
```